scripts/build_android.sh

```diff
@@ -102,8 +102,8 @@ if [ -z "$BUILD_MOBILE_TEST" ]; then
 fi
 # Don't build artifacts we don't need
 CMAKE_ARGS+=("-DBUILD_TEST=OFF")
-CMAKE_ARGS+=("-DC10_MOBILE=TRUE")
 CMAKE_ARGS+=("-DBUILD_BINARY=OFF")
+CMAKE_ARGS+=("-DBUILD_LITE_INTERPRETER=ON")
```

Comment:
Eventually this flag should be optional. I'd suggest to have an env variable, ```if ["$BUILD_LITE_INTREPRETER" ]; then CMAKE_ARGS+=("-DBUILD_LITE_INTREPRETER=ON")```
We could run 
```BUILD_LITE_INTREPRETER=1 build_android.sh``` and ```BUILD_LITE_INTREPRETER=1 build_pytorch_android.sh``` when build lite interpreter. 
Without ```BUILD_LITE_INTREPRETER=1``` the scripts are still working for full-jit.

Reply:
It's a great idea to use env variable! I update if-statement logic `if ["$BUILD_LITE_INTREPRETER" == 1 ]`. It seems like `if ["$BUILD_LITE_INTREPRETER" ]` will keep being true when running `BUILD_LITE_INTREPRETER=1 build_android.sh` then `build_android.sh`